test/integration/multikueue/multikueue_test.go

```diff
@@ -1404,7 +1404,7 @@ var _ = ginkgo.Describe("Multikueue", ginkgo.Ordered, ginkgo.ContinueOnFailure,
 				// The transition interval should be close to testingKeepReadyTimeout (taking into account the resolution of the LastTransitionTime field)
 				g.Expect(acs.LastTransitionTime.Time).To(gomega.BeComparableTo(disconnectedTime.Add(testingWorkerLostTimeout), cmpopts.EquateApproxTime(2*time.Second)))
 
-				g.Expect(createdWorkload.Status.Conditions).To(utiltesting.HaveConditionStatusTrue(kueue.WorkloadQuotaReserved))
+				g.Expect(createdWorkload.Status.Conditions).To(utiltesting.HaveConditionStatusFalse(kueue.WorkloadQuotaReserved))
```

Comment:
Please add a comment / explanation why this is changed

Reply:
This was actually the state of the test before this [PR](https://github.com/kubernetes-sigs/kueue/pull/3348/files#diff-324526e93262c68c8d44636dca0fbbcb072ae3728e8146e1421d70596a9be647) which I believe was a bug

Reply:
```suggestion
				g.Expect(createdWorkload.Status.Conditions).ToNot(utiltesting.HaveConditionStatusTrue(kueue.WorkloadQuotaReserved))
```

Oops. My bad.

Reply:
I can fix it in a separate PR if you give me the chance.

Reply:
Sure let's decouple the fix if possible

Reply:
Done on https://github.com/kubernetes-sigs/kueue/pull/3356.

Reply:
@PBundyra Could we revert this change? We will fix this in a separate PR.

Reply:
Sure, done. Thank you!